src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -749,6 +749,9 @@ private static String processPageSourceForGodMode(String content) {
         return content
                 .replaceAll("<#comment[ ]*</#comment>", "<!---->")
                 .replace(Config.APP_URL, "${app.url}")
+                .replace("http://localhost:8888", "${web.url}")
+                .replace(Config.APP_URL.replace("http", "https"), "${web.url}")
+                .replace("\"/_ah", "\"${web.url}/_ah")
```

Comment:
"_ah/" happens in dev while the complete url is shown in the staging server's logout link. However, since most of us fix html tests in dev server, we only see "_ah/" in the links (explains why you did not see the "${web.url}" happen when you reloaded the html else where)